plugins/module_utils/version.py

```diff
@@ -10,4 +10,41 @@
 __metaclass__ = type
 
 
+import operator
+from functools import wraps
+
 from ansible.module_utils.compat.version import LooseVersion  # noqa: F401, pylint: disable=unused-import
+
+
+def _version_compare(op):
+    @wraps(op)
+    def _op(a, b):
+        return op(LooseVersion(a), LooseVersion(b))
+    return _op
+
+
+version_ops = {
```

Comment:
I think it's important to make clear that this is about `LooseVersion`. There's also `StrictVersion`, which is what is used for Python package versions for example.
```suggestion
loose_version_ops = {
```